recipes/r-dotcall64/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = '0.9-04' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-dotcall64
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: dotCall64_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/dotCall64_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/dotCall64/dotCall64_{{ version }}.tar.gz
+  sha256: 0e65a56c836aecd02380d9033989a05e391552eb6b3a9d16504a827b9faae662
+
+build:
+  number: 0
+  script: R CMD INSTALL --build .
+  skip: True  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [linux]
+
+  run:
+    - r-base
+    - libgcc               # [linux]
```

Comment:
You need to add `libgfortran` for `OS X`.

> Actually, unless we are targeting "headless" machines we don't even need it on Linux.

That's not true. Since we are using the gcc from the conda package here, if `libgcc` is not added, there's a chance that the package will not work in CentOS 6 (which is the OS of the docker container we use).